docs/develop/dapps/payment-processing/accept-payments-in-a-telegram-bot-js.md

```diff
@@ -0,0 +1,440 @@
+# Bot for the sale of dumplings
+
+
+In this article, we'll create a simple Telegram bot for accepting payments in TON coins.
+
+## 🦄 What it looks like
+
+The bot will look like this:
+
+![bot preview](https://raw.githubusercontent.com/ton-community/ton-docs/main/static/img/tutorials/js-bot-preview.png?raw=true)
+
+### Source code
+
+Sources are available on GitHub:
+* https://github.com/coalus/DumplingShopBot
+
+## 📖 What you'll learn
+You'll learn how to:
+ - Create a Telegram bot in NodeJS using Grammy
+ - Work with public TON API
+
+## ✍️ What you need to get started
+Install [NodeJS](https://nodejs.org/en/download/) if you haven't yet.
+
+Also you need these libraries:
+ - grammy
+ - ton
+ - dotenv
+
+You can install them with one command in the terminal.
+```bash
+npm install
```

Comment:
Instead of a list of libs better to use the command:
```
npm install <lib1> <2> <3>...
```

+ **secret hack!** add `npm2yarn` word after `bash` text in code block. It will automatically convert format for npm/yarn. 

Example: https://www.tonspace.co/develop/smart-contracts/sdk/javascript